src/CSharpRestApiClientKeystone.Analyzers/ReadAsStringInUseAsyncAnalyzer.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Immutable;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace CSharpRestApiClientKeystone.Analyzers
```

Comment:
I think that having implemented a second analyzer is the reason why you're getting two analysis results here.
I'm not sure what the best practice is when designing analyzers for problem type to analyzer.
But if we look at xunit as an example, they seem to be grouping all the same problem types in a single analyzer.
This one looks at all the IEnumerable variations in a single analyzer.
https://github.com/xunit/xunit.analyzers/blob/98d4cd744bc52999068c10c0a70fd0e28d15d554/src/xunit.analyzers/X1000/ClassDataAttributeMustPointAtValidClass.cs#L16
What do you think?